README.old.md

```diff
@@ -0,0 +1,73 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
```

Comment:
- [ ] Let's just keep one of the two README's. In this case, I recommend keeping the one from Microverse. You can use the relevant information from the one generated by CRA to keep this one updated. Remember that keeping our repos professional at all times it's important because recruiters might check them at any moment.

You can read more about to keep the [repository professional here.](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)